### PR TITLE
Permissions rename

### PR DESCRIPTION
**Pull request by ccfredbaum**
Rename permissions - fix shortcodes.

### CODE REVIEW

File: my-video-room/views/admin/defaultsettings.php [hunk 1]
```diff
@@ -13,28 +13,30 @@
...
+	if ( $tab_count >= 1 ) {
+		?>
+<h2><?php esc_html_e( 'MyVideoRoom Default Settings', 'my-video-room' ); ?></h2>
+<p><?php esc_html_e( 'This section allows you manage the default room appearance as well as permissions, guest/host decisions, and room security settings across all of your rooms.', 'myvideoroom' ); ?>
 </p>
 <nav class="myvideoroom-nav-tab-wrapper nav-tab-wrapper">
 	<ul>
 		<li>
-			<a class="nav-tab nav-tab-active" href="#defaulthost">
+			<a class="nav-tab nav-tab-active" href="#defaulthost<?php echo esc_attr( $html_library->get_id( $string_randomizer_input ) ); ?>">
```
Comment (alecsammon) [inline my-video-room/views/admin/defaultsettings.php:39]: You can remove the `defaulthost` from the beginning of this - the `get_id` method adds it

File: my-video-room/views/admin/defaultsettings.php [hunk 2]
```diff
@@ -13,28 +13,30 @@
...
  *
  * @param array $all_wp_roles A list of WordPress roles. @see global $wp_roles->roles.
  */
 return function (
 	array $all_wp_roles = array()
 ): string {
 	\ob_start();
+	$string_randomizer_input = 'defaulthosts';
+	$html_library            = Factory::get_instance( HTML::class, array( $string_randomizer_input ) );
+	$html_lib                = $html_library;
```
Comment (alecsammon) [inline my-video-room/views/admin/defaultsettings.php:26]: not sure why we are reassigning this variable.

File: my-video-room/views/admin/defaultsettings.php (hunk 1 above)
Comment (ccfredbaum) [reply my-video-room/views/admin/defaultsettings.php:39]: great- will do

File: my-video-room/views/admin/defaultsettings.php (hunk 2 above)
Comment (ccfredbaum) [reply my-video-room/views/admin/defaultsettings.php:26]: me neither- will remove

**Pull request merged**